src/Congo.WebApi/Startup.cs

```diff
@@ -29,6 +31,10 @@ public void ConfigureServices(IServiceCollection services)
                 c.SwaggerDoc("v1", new OpenApiInfo { Title = "Congo.WebApi", Version = "v1" });
             });
             services.AddMediatR(typeof(Startup).Assembly);
+
+            var config = new TypeAdapterConfig();
```

Comment:
Is this included for future use? We currently don't really need a config, so it could be left for when we need it, in accordance with YAGNI and KISS :)

Reply:
wdym? :0

Reply:
I've only used Mapsters extension methods before, not injecting the mapper. Either way works fine really, and for the mapper approach this is mandatory it seems, so ignore this comment :)